cdc/cdc/owner/changefeed.go

```diff
@@ -73,38 +75,43 @@ type changefeed struct {
 	metricsChangefeedResolvedTsGauge      prometheus.Gauge
 	metricsChangefeedResolvedTsLagGauge   prometheus.Gauge
 
-	newDDLPuller func(ctx cdcContext.Context, startTs uint64) (DDLPuller, error)
-	newSink      func() DDLSink
+	// newDDLPuller func(ctx cdcContext.Context, startTs uint64) (DDLPuller, error)
+	// newSink      func() DDLSink
 	newScheduler func(ctx cdcContext.Context, startTs uint64) (scheduler, error)
 }
 
 func newChangefeed(id model.ChangeFeedID, gcManager gc.Manager) *changefeed {
 	c := &changefeed{
 		id: id,
 		// The scheduler will be created lazily.
-		scheduler:        nil,
-		barriers:         newBarriers(),
+		scheduler: nil,
+		// barriers:         newBarriers(),
 		feedStateManager: new(feedStateManager),
 		gcManager:        gcManager,
 
 		errCh:  make(chan error, defaultErrChSize),
 		cancel: func() {},
 
-		newDDLPuller: newDDLPuller,
-		newSink:      newDDLSink,
+		// newDDLPuller: newDDLPuller,
```

Comment:
ditto

Reply:
In TiCDC, owner is responsible for two things:
1. Divide changfeed into tasks and assign them to other captures.
2. Synchronize DML operations of upstream TiDB.
Now synchronizing DML operations don't need.